test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -252,7 +256,7 @@ export default class BaseFormObject {
 
   getTextBoxErrorMessage(fieldName) {
     if (userVariables.v3) {
-      return this.el.find(`#${fieldName}-error`).innerText;
```

Comment:
I have no idea why I had to change this, I have seen ti work like this on other branches, but here, it continued to fail locally until I changed it to use the `[id=""]` selector. Any ideas?